cypress.env.json

```diff
@@ -0,0 +1,3 @@
+{
+  "source": "https://github.com/electric-cash/explorer"
```

Comment:
I don't think that this URL changes depending on the environment. If there are no many benefits in keeping it in a JSON file I'd move it to the code instead.